src/dotnet/commands/dotnet-build3/ForwardingApp.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli
+{
+    /**
```

Comment:
Thanks for adding the comments.

Now for my nits:  C# summary comments use the following format: `/// <summary>`